### ISSUE TITLE
Questions about reproducing OV-COCO results

### ISSUE THREAD

**Issue by Parsifal133** [CVMI-Lab/CoDet#11]
Thank you for your awesome work and open source code!

 I'm trying to reproduce the OV-COCO experiment, the results in Table 2 in your paper.

 Unfortunately, there is only one 1080Ti available, so I divided the base learning rate by 8 (as mentioned in the repo), but the results are quite different from the data in your paper （The experimental results are shown in the figure below, unseen AP50=19, seen AP50=50）. 

I tried doubling the training iterations, and only got unseen AP50=24, seen AP50=50.

Can you provide some suggestions?

![447e9b10eb1f2d3e1e9add904d0cefd](https://github.com/CVMI-Lab/CoDet/assets/55016754/67855aab-3e97-4100-9588-eec8b7a18d65)

**Comment by machuofan** [comment]
Thanks for your interest in our work!

CoDet mixes up detection data and image-text pairs during training. That is, if you use 8 GPUs for training, you would probably have 4 GPUs loaded with detection data and 4 GPUs loaded with image-text pairs. But since you use 1 GPU for training, you can only have one type of data loaded in each iteration, which dominates the gradients and results in unstable training. 

Therefore, to reproduce the results in our paper, I suggest using at least 4 GPUs for training.

**Comment by Parsifal133** [comment]
Thanks for your detailed reply, I will try more GPUs.
